bio/emu/abundance/wrapper.py

```diff
@@ -0,0 +1,30 @@
+__author__ = "Curro Campuzano Jimenez"
+__copyright__ = "Copyright 2024, Curro Campuzano Jimenez"
+__email__ = "campuzanocurro@gmail.com"
+__license__ = "MIT"
+
+
+from snakemake.shell import shell
+import tempfile
+import os
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+extra = snakemake.params.get("extra", "")
+if db := snakemake.input.get("db"):
```

Comment:
So it is compatible with py39.
```suggestion
db = snakemake.input.get("db", ""):
if db:
```